io.catenax.manufacturing_capability/3.0.0/ManufacturingCapability.ttl

```diff
@@ -0,0 +1,303 @@
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#>.

```

Comment:
copyright headers are missing

Reply:
Done in https://github.com/eclipse-tractusx/sldt-semantic-models/pull/574/commits/e65ddddf876318f0512a3156859223fecc99b912